### DISCUSSION TITLE
J4 RC5 - Default Template Name to Site name?

### DISCUSSION THREAD

**Discussion by coolcat-creations** [joomla/joomla-cms#47131] [2. Cold Features]
### What needs to be fixed
A fresh Joomla installations shows, whatever Sitename you enter the cassiopeia title in frontend. I see no sense in naming the site cassiopeia - In Joomla 3 the Site Logo was the Site Name

### Why this should be fixed
It improves the user experience and the user will be happy to see his / hers Site title already on the Frontend after installation.

### How would you fix it
Insert the site title in the template field in the installation process

### Side Effects expected
Only if the site-title is very long, but that was the same in J3

**Comment by brianteeman** [comment]
I agree but I fear that after being ignored for 4 years it is now too late to fix
